extensions/todolist@bsaleil.org/Makefile.am

```diff
@@ -0,0 +1,14 @@
+todoschemadir = $(extensiondir)/schemas
```

Comment:
I was wondering if we shouldn't just patch this extension to use the default schema directory in case when `<todo extension directory>/schemas` subdirectory is missing (like time tracker is doing).